vfs/vfscache/item.go

```diff
@@ -399,9 +402,37 @@ func (item *Item) IsDirty() bool {
 	return item.metaDirty || item.info.Dirty
 }
 
+// IsDataDirty returns true if the item's data is dirty
+func (item *Item) IsDataDirty() bool {
+	item.mu.Lock()
+	defer item.mu.Unlock()
+	return item.info.Dirty
+}
+
+// Open the local file from the object passed in.  Wraps open()
+// to provide recovery from out of space error.
+func (item *Item) Open(o fs.Object) (err error) {
+	for {
```

Comment:
This probably shouldn't retry forever...

Reply:
Use a maximum retry count of 20 for this commit and potentially add a timeout in the future?

Reply:
Probably `fs.Config.LowLevelRetries` is the right number (which defaults to 10)